### PR TITLE
Fixes #502

### LINKED ISSUE #502: Queue.enqueue_call() ignores ttl
Although `Queue.enqueue_call()` accepts a `ttl` parameter, it doesn't pass it to `self.job_class.create()`

### PR DESCRIPTION
**Pull request by mbodock**
Fixes some broken tests and misbehaviour with ttls. There was a temporal
coupling between saving the job and setting its expires parameter.

### CODE REVIEW

**Comment by herberthamaral** [comment]
(y)

File: rq/job.py [hunk 1]
```diff
@@ -417,6 +417,7 @@ def to_date(date_str):
         self.result_ttl = int(obj.get('result_ttl')) if obj.get('result_ttl') else None  # noqa
         self._status = as_text(obj.get('status') if obj.get('status') else None)
         self._dependency_id = as_text(obj.get('dependency_id', None))
+        self.ttl = int(obj.get('ttl', -1))
```
Comment (selwin) [inline rq/job.py]: Thanks for the pull request! The default `ttl` should be `None` and not `-1`.

**Comment by selwin** [comment]
@mbodock this PR also no longer merges cleanly. It would be great if you can update this sometime in the next few days. I'd like to include this fix in the upcoming 0.5.3 release :)

**Comment by mbodock** [comment]
@selwin Done.

**Comment by selwin** [comment]
Thanks!

**Pull request merged**